src/LanguageHandler/Php/Parser/Entity/BaseEntity.php

```diff
@@ -412,7 +527,7 @@ private function fillInLinkDataWithUrls(array $linkData): array
                 }
 
                 if ($entityData['entityName']) {
-                    $linkData[$key]['url'] = call_user_func_array(
+                    $url = call_user_func_array(
```

Comment:
PHPStorm is complaining about this style of usage with:
>[EA] 'call_user_func($this->documentedEntityUrlFunction, $this->getRootEntityCollection(), $entityData['entityName'], $entityData['cursor'])' would make possible to perform better code analysis here.

It seems to think the following would be better:
```
                    $url = call_user_func(
                        $this->documentedEntityUrlFunction,
                        $this->getRootEntityCollection(),
                        $entityData['entityName'],
                        $entityData['cursor']
                    );
```

Reply:
Done